CHANGELOG.md

```diff
@@ -3,6 +3,37 @@ Changelog for NeoFS Node
 
 ## [Unreleased]
 
+## [0.24.0] - 2021-08-30 Anmyeondo (안면도, 安眠島)
+
+### Fixed
+- Liter warning messages (#766)
+- Storage Node does not register itself in network in relay mode now (#761)
+
+### Changed
+- `neofs-adm` fails when is called in a notary-disabled environment (#757)
+- `neofs-adm` uses `neo-go` client's native NNS resolving method instead of the custom one (#756)
+- Node selects pseudo-random list of objects from metabase for replication (#715)
+
+### Added
+- Contract update support in `neofs-adm` utility (#748)
+- Container transferring support in `neofs-adm` utility (#755)
+- Storage Node's balance refilling support in `neofs-adm` utility (#758)
+- Support `COMMON_PREFIX` filter for object attributes in storage engine and `neofs-cli` (#760) 
+- Node's and IR's notary status debug message on startup (#758)
+- Go `1.17` unit tests in CI (#766)
+- Supporting all eACL filter fields from the specification (#768)
+- Cache for Container service's `List` operation (#676)
```

Comment:
It's more like cache for all Neo RPC readonly operations in container service.

Reply:
changed